test/authentication_tests.coffee

```diff
@@ -50,7 +50,9 @@ Meteor.startup ->
 
         next()
 
-    it 'should not allow a user with wrong password to login', (test, next) ->
+    it 'should not allow a user with wrong password to login and should respond after 500 msec', (test, next) ->
```

Comment:
I combined the two tests into one primarily because the failed response now takes at least 0.5 seconds. If I test the delay separately, we will add another second, which wasn't really necessary...